modules/wkt/src/lib/utils/hex-transcoder.ts

```diff
@@ -0,0 +1,50 @@
+// Forked from https://github.com/jessetane/hex-transcoder under MIT license
+
+const alphabet = '0123456789abcdef';
+const encodeLookup: string[] = [];
+const decodeLookup: number[] = [];
+
+for (let i = 0; i < 256; i++) {
+  encodeLookup[i] = alphabet[(i >> 4) & 0xf] + alphabet[i & 0xf];
+  if (i < 16) {
+    if (i < 10) {
+      decodeLookup[0x30 + i] = i;
+    } else {
+      decodeLookup[0x61 - 10 + i] = i;
+    }
+  }
+}
```

Comment:
Some comments:
1. alphabet is already a lookup table, so there is no need to create encodeLookup/decodeLookup. 
2. alphabet = '0123456789abcdef', so it's better to make sure the input/output string are converted to lowercase

Reply:
Hmm, good point. Let me get back to that in another PR. Ideally this function should be quite fast so we should be smart about it.